packages/halo/src/react/index.tsx

```diff
@@ -16,19 +16,12 @@ import * as vars from '../vars'
 
 if (canUseDOM()) polyfillFocusWithin(document)
 
-type StyleFn = (
+export type StyleFn<P> = (
```

Comment:
It'd be good to put this in util or somewhere common.

Would allow reuse.

Would also help us to be consistent in the format of these things.

But, maybe we don't have consistent need.  For example, some StyleFn's include state as a param. This one obviously doesn't need that.

Reply:
yeah i'm torn. i'm not sure all our styles fns can be the same - like when we have multiple components in a single file that have different props